### ISSUE TITLE
Typo in Screen Reader script has typo where 7 is said instead of at

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/ratio-and-proportion#310]
**Test device**
Dell and iPad 6th Gen
**Operating System**
Win 10 and iPadOS 14.3
**Browser**
Chrome and Safari
**Problem description**
For https://github.com/phetsims/QA/issues/582. First seen in a11y view on Windows, confirmed when read out on mobile VO.
In several instances where the screen reader should read out that the hands are "at challenge ratio" (I believe that is what it means at least) it instead says "7 challenge ratio." This is very consistent, showing up on both screens at different points of the description and in the alerts. If the 7 is meant to be there and means something I can't figure out what it could be.

**Visuals**
<img width="288" alt="more7ratio" src="https://user-images.githubusercontent.com/41024075/102391511-0ca7d480-3f93-11eb-9b3f-96e71727fc57.png">
<img width="878" alt="handsare7" src="https://user-images.githubusercontent.com/41024075/102391514-0d406b00-3f93-11eb-9df7-7332c9d7e0d0.png">

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪Ratio and Proportion‬
URL: https://phet-dev.colorado.edu/html/ratio-and-proportion/1.0.0-dev.77/phet/ratio-and-proportion_en_phet.html?postMessageOnLoad&postMessageOnError&supportsDescriptions
Version: 1.0.0-dev.77 2020-12-08 00:32:10 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.88 Safari/537.36
Language: en-US
Window: 1280x658
Pixel Ratio: 1.5/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by zepumph** [comment]
Yeah sorry, that was fixed by https://github.com/phetsims/ratio-and-proportion/commit/9892edd10b5e19764c755d6953055ac74b326695, two commits after this dev version was created.
